src/services/hooks/useLatest.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { MutableRefObject, useRef } from 'react';
+
+/**
+ * Wraps the given `value` into a `MutableRefObject` and keeps the `current`
+ * value up-to-date on very render cycle.
```

Comment:
Super minor copy nit, also optional - feel free to merge w/o
```suggestion
 * value up-to-date on every render cycle.
```